x/concentrated-liquidity/tick.go

```diff
@@ -81,15 +81,23 @@ func (k Keeper) SetTickInfo(ctx sdk.Context, poolId uint64, tickIndex int64, tic
 // Also, lower tick must be less than upper tick.
 // Returns error if validation fails. Otherwise, nil.
 // TODO: test
-func validateTickRangeIsValid(lowerTick int64, upperTick int64) error {
-	// ensure types.MinTick <= lowerTick < types.MaxTick
+func validateTickRangeIsValid(tickSpacing uint64, lowerTick int64, upperTick int64) error {
+	// Check if the lower and upper tick values are divisible by the tick spacing.
+	if lowerTick%int64(tickSpacing) != 0 || upperTick%int64(tickSpacing) != 0 {
```

Comment:
Just wondering - what if tick spacing is say 3 or 6, will users have to think about multiples of these numbers when LPing? Is there a way to avoid this UX impediment?

Reply:
The solution is to make specific ticks (1,10,30,60,200) as uni does rather than a range